crates/rooch/src/commands/move_cli/commands/new.rs

```diff
@@ -21,13 +21,25 @@ const ROOCH_FRAMEWORK_PKG_PATH: &str = "{ git = \"https://github.com/rooch-netwo
 
 #[derive(Parser)]
 pub struct New {
+    /// Existing account address from Rooch
+    #[clap(long = "address", short = 'a')]
+    account_address: Option<AccountAddress>,
+
     #[clap(flatten)]
     pub new: new::New,
 }
 
 impl New {
     pub fn execute(self, path: Option<PathBuf>) -> anyhow::Result<()> {
         let name = &self.new.name.to_lowercase();
+        let address = if let Some(account_address) = &self.account_address {
+            // Existing account address is available
+            account_address.to_hex_literal()
+        } else {
+            // Existing account address is not available, generate a new random address
+            AccountAddress::random().to_hex_literal()
```

Comment:
if the account address is None, use the default account.

Reply:
The account is now the default account from config.